### ISSUE TITLE
Recommendation: Avoid duplicate translation keys

### ISSUE THREAD

**Issue by jniles** [third-culture-software/bhima#191]
Our translation files always seem to be expanding.  I've noticed lots of additions in PRs coming into the repository.  You can also see for yourself, just run this command (linux):

``` sh
less client/src/i18n/fr.json | grep "UPDATE_SUCCES"
```

You should see the following output:

``` sh
   "UPDATE_SUCCESS_MESSAGE" : "Mise à jour avec succès !"
   "UPDATE_SUCCESS_MESSAGE" : "Mise à jour avec succès !"
    "UPDATE_SUCCESS"         : "Mis à jour une caisse avec succès."
   "UPDATE_SUCCES"        : "Mis à jour avec succès"
   "UPDATE_SUCCES"        : "Mis à jour avec succès"
   "UPDATE_SUCCES"     : "Mis à jour avec succès"
   "UPDATE_SUCCES"  : "Mis à jour avec succès"
   "UPDATE_SUCCES"  : "Mise à jour avec succès"
  "UPDATE_SUCCESS"    : "Mise a jour du depot avec succès"
   "UPDATE_SUCCES"  : "Mis à jour avec succès"
   "UPDATE_SUCCESS"     : "Mis à jour avec succès"
   "UPDATE_SUCCES"            : "Mis à jour avec succès"
   "UPDATE_SUCCES"  : "Mis à jour avec succès"
   "UPDATE_SUCCES"   : "Mis à jour avec succès",
   "UPDATE_SUCCES"  : "Mis à jour avec succès",
   "UPDATE_SUCCES"  : "Mis à jour avec succès"
      "UPDATE_SUCCESS" : "Item modifié!"
   "UPDATE_SUCCES"  : "Mis à jour avec succès"
   "UPDATE_SUCCESS" : "Patient modifier avec succès",
   "UPDATE_SUCCES"  : "Mis à jour avec succès",
   "UPDATE_SUCCESS" : "Fait un mis a jour avec success.",
   "UPDATE_SUCCES"  : "Mis à jour avec succès"
   "UPDATE_SUCCES"   : "Mis à jour avec succès",
   "UPDATE_SUCCES"   : "Mis à jour avec succès",
   "UPDATE_SUCCES"  : "Mis à jour avec succès",
   "UPDATE_SUCCES"          : "Mis à jour avec succès"
   "UPDATE_SUCCESS" : "Mise à jour avec succès"
   "UPDATE_SUCCES"       : "Mise à jour avec succès"
   "UPDATE_SUCCESS" : "Mise à jour avec succès"
```

That is a lot of repeated phrases that are hard to maintain, wasted space when shipped to the client, difficult to find and edit, and overall just poor programming practice.

**Proposal**
Someone should create form translation keys that can be used by any module to convey generic messages such as "created", "updated", "errored", etc.  It would look like this:

``` html
<span class="text-success">{{ 'FORM.UPDATED' | translate }}</span>
<span class="text-success">{{ 'FORM.CREATED' | translate }}</span>
<span class="text-danger">{{ 'FORM.ERRORED' | translate }}</span>
<span class="text-warning">{{ 'FORM.DELETED' | translate }}</span>
```

With just this simply consolidation, we can probably eliminate > 100 lines from our translation files.  
@DedrickEnc , would you review this proposal and assign it to someone if you think it is a good idea?

**Comment by DedrickEnc** [comment]
We planed to design the translation file entirely, do you think it is not
worth to wait until we will finish financiary module development?
If not, I will do it!

**Comment by jniles** [comment]
If we fix this sooner, the less work we will have to do later.  It is similar to building components first, then using them in the module.

I say we do it sooner rather than later!

**Comment by jniles** [comment]
If we fix this sooner, the less work we will have to do later.  It is similar to building components first, then using them in the module.

I say we do it sooner rather than later!

**Comment by jniles** [comment]
This was closed via #197 and #283.
